blocks/gallery/gallery.css

```diff
@@ -1 +1,34 @@
-/* empty for now */
+/* carousel 3rd party styles */
+/* stylelint-disable selector-class-pattern */
+
+/* revert global styles from button */
```

Comment:
@tamara027 we need to revert all the styles manually which have been setup as global styles here. Without changing the solution we have in place, I can see no other way.